manim/mobject/types/vectorized_mobject.py

```diff
@@ -1192,10 +1205,11 @@ def rotate(
         angle: float,
         axis: Vector3D = OUT,
```

Comment:
This one would probably also be autofixed:
```suggestion
        axis: Vector3DLike = OUT,
```